applications/FemToDemApplication/custom_elements/base_solid_element.h

```diff
@@ -875,8 +876,30 @@ class KRATOS_API(FEM_TO_DEM_APPLICATION) BaseSolidElement
     */
     void CalculateShapeGradientOfMassMatrix(MatrixType& rMassMatrix, ShapeParameter Deriv) const;
 
-    double GetRayleighAlpha(const Properties& rProperties, const ProcessInfo& rCurrentProcessInfo);
-    double GetRayleighBeta(const Properties& rProperties, const ProcessInfo& rCurrentProcessInfo);
+    /**
+     * @brief Get the Rayleigh alpha parameter.
+     * @details This function calculates and returns the Rayleigh alpha parameter based on the provided properties and current process information.
+     * @param rProperties The properties needed for the calculation.
+     * @param rCurrentProcessInfo The current process information.
+     * @return The Rayleigh alpha parameter.
+     */
+    double GetRayleighAlpha(
+        const Properties& rProperties,
```

Comment:
did you intend to change the elements from the `FemToDemApp`?

Reply:
By error :(